pytket/docs/changelog.rst

```diff
@@ -1,15 +1,19 @@
 Changelog
 =========
 
+1.6.0 (September 2022)
+----------------------
+
+* New ``ToffoliBox`` for constructing circuits that implement permutations of
+  basis states.
+
 1.5.2 (August 2022)
 -------------------
 
 Minor new features:
 
 * Prefer `ZZPhase` in ``DecomposeTK2`` if it results in the same fidelity but
   fewer two-qubit gates.
-* New ``ToffoliBox`` for constructing circuits that implement permutations of
```

Comment:
Is it intended to remove this?

Reply:
It should not have been under 1.5.2 as it was introduced since then.